graviti/dataframe/frame.py

```diff
@@ -151,43 +152,27 @@ def _get_repr_header(flatten_header: List[Tuple[str, ...]]) -> List[List[str]]:
         return lines
 
     @classmethod
-    def _construct(
-        cls,
-        columns: Dict[str, Union["DataFrame", ColumnSeries]],
-    ) -> "DataFrame":
+    def _construct(cls, columns: Dict[str, Container]) -> "DataFrame":
         obj: DataFrame = object.__new__(cls)
         obj._columns = columns
         obj._column_names = list(obj._columns.keys())
         return obj
 
     @classmethod
-    def _from_pyarrow(cls: Type[_T], array: pa.StructArray, schema: _S) -> _T:
+    def _from_pyarrow(cls: Type[_T], array: pa.StructArray, schema: pt.PortexType) -> _T:
         obj: _T = object.__new__(cls)
         obj.schema = schema
         obj._columns = {}
         obj._column_names = []
 
-        # In this case it's always record.
-        expanded_schema: record = schema.to_builtin()  # type:ignore[assignment]
-
-        for pafield in array.type:
-            column_name = pafield.name
-            column_value = array.field(column_name)
-            column_schema = expanded_schema.fields[column_name]
-            if isinstance(pafield.type, pa.StructType):
-                # if pafield.type is pa.StructType,
-                # the schema will always be record or PortexExternalType.
-                obj._columns[column_name] = DataFrame._from_pyarrow(
-                    column_value, schema=column_schema  # type:ignore[arg-type]
-                )
-            else:
-                obj._columns[
-                    column_name
-                ] = ColumnSeries._from_pyarrow(  # pylint: disable=protected-access
-                    column_value, schema=column_schema
-                )
+        # In this case schema.to_builtin() should always record.
```

Comment:
`In this case schema.to_builtin() always returns record`?

Reply:
if the check in `DataFrame.from_pyarrow` passed, the `schema.to_builtin` should always be `record`

Reply:
I mean should we rephrase the sentence to the previous comment?